src/app/api/auth/token/route.ts

```
@@ -0,0 +1,27 @@
+        });
+
+        // Handle missing token
+        if (!token?.accessToken) {
+            return NextResponse.json({ accessToken: null }, { status: 401 });
+        }
+
+        // Return the access token
+        return NextResponse.json({ accessToken: token.accessToken });
+    } catch (error) {
+        // Handle error
+        console.error("Error fetching token:", error);
```

ESLint

[error] Unexpected console statement.

(no-console)